app/src/main/java/net/skyscanner/backpack/demo/stories/BarChartStory.kt

```diff
@@ -40,14 +40,15 @@ class BarChartStory : Story() {
           createMonth(5)
         ),
         legend = BpkBarChart.Legend(
-          "£",
-          "No Price"
+          selectedTitle = "Selected",
+          inactiveTitle = "No Price",
+          activeTitle = "Price",
         )
       )
     }
   }
 
-  private val random = Random()
+  private val random = Random(18735)
```

Comment:
no more changing barchart screenshots!

Reply:
But why 18735?

Reply:
Tried a few random numbers until I got one that had a nice mix of values

Reply:
What about 42? It should answer anything